server/server.js

```diff
@@ -1,7 +1,105 @@
-const app = require('./app.js');
+require('newrelic');
+const port = process.env.PORT || 8082;
+const Promise = require('bluebird');
+const path = require('path');
+const fs = Promise.promisifyAll(require('fs'));
+const http = require('http');
+const mongoose = require('mongoose');
+const redisClient = require('redis').createClient;
 
-const port = process.env.PORT || 8081;
+const redis = redisClient(6379, 'localhost');
 
-const server = app.listen(port);
+const db = mongoose.createConnection('mongodb://localhost/restaurant');
 
-module.exports = server;
+const About = db.model('About', new mongoose.Schema({
+  id: {
+    type: Number,
+    unique: true,
+  },
+  name: String,
+  about: {
+    description: String,
+    hours: String,
+    price: String,
+    style: String,
+    phone: String,
+  },
+  banner: [],
+  photo: [],
+}));
+// let MongoCount = 0;
+// let RedisCount = 0;
+http.createServer((req, res) => {
+  const { method, url } = req;
+  if (url === '/' || url.startsWith('/?=')) {
+    fs.readFileAsync(path.join(__dirname, '../client/dist/index.html'), 'utf8')
+      .then((data) => {
+        res.writeHead(200, { 'Content-Type': 'text/html' });
+        res.end(data);
+      })
+      .catch(() => {
+        res.writeHead(500);
+        res.end();
+      });
+  } else if (url.match('.js')) {
```

Comment:
I think this would match a request for any `.js` file with the `bundle.js` file?  Could you parse out the requested filename and serve that?